### ISSUE TITLE
Roboland: Map icon

### ISSUE THREAD

**Issue by Snook-8** [jchmb/dst-multiverse#94]
Doesnt exist, shows nothing at the map at the spot that the portal exists.

**Comment by YiinHikari** [comment]
It only happen with Roboland icon. I tested in bunnyland, greyland and snowland for now. Will finish tomorrow. But i am pretty sure it will happen in the other worlds aswell. I think we should replace all words "chezz" with "robo" in the code and unify this because I think that is what causes the problem (I use sublime text 2 and is pretty easy to find all the chezz words in all the mod's code to replace them. If there is an easy way, you tell me :) And after replace them just erase the duplicated minimap icon with the chezz name.. In the other hand, all the portals I saw by now says the destination correctly.

![20180901010022_1](https://user-images.githubusercontent.com/25496851/44942380-cdcfbf80-ad85-11e8-8008-e049ea37146a.jpg)
![20180901003642_1](https://user-images.githubusercontent.com/25496851/44942384-eb9d2480-ad85-11e8-912d-15221c6546d1.jpg)
![20180901001635_1](https://user-images.githubusercontent.com/25496851/44942394-243cfe00-ad86-11e8-91ff-0ed416b7f5fa.jpg)
